### PR TITLE
Group parameters for RDGW templates

### PR DESCRIPTION
**Pull request by dvu70**
-tripleaz
-dualaz
-singleaz

### CODE REVIEW

File: templates/ra_dualaz_rdgw_elb.compound [hunk 1]
```diff
@@ -123,6 +123,83 @@
...
+            "ParameterLabels" : 
+            {
+                "RdgwAmiNameSearchString" : 
+                { 
+                    "default" : "RDGW AMI Search Pattern"
+                }
+            }    
+        }
+    },
+
```
Comment (lorengordon) [inline templates/ra_dualaz_rdgw_elb.compound]: Extra blank line?

File: templates/ra_singleaz_rdgw_elb.compound [hunk 2]
```diff
@@ -118,6 +118,81 @@
...
+            "ParameterLabels" : 
+            {
+                "RdgwAmiNameSearchString" : 
+                { 
+                    "default" : "RDGW AMI Search Pattern"
+                }
+            }    
+        }
+    },
+
```
Comment (lorengordon) [inline templates/ra_singleaz_rdgw_elb.compound]: Unnecessary blank line?

**Comment by dvu70** [comment]
Removed extra space for dualaz, singleaz

**Comment by dvu70** [comment]
Meant line

**Comment by lorengordon** [comment]
lgtm :ship:

**Pull request merged**